test/benchmarks/benchmark_crc32.cc

```
@@ -3,59 +3,91 @@
  * For conditions of distribution and use, see copyright notice in zlib.h
  */
 
-#include <stdio.h>
-#include <assert.h>
-
 #include <benchmark/benchmark.h>
+#include <assert.h>
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)